oyez_brief.py

```diff
@@ -43,15 +58,26 @@ def oyez_get(approx_term, docket_number):
     return None
 
 
-def oyez_brief(*args):
-    og = oyez_get(*args)
+def _oyez_brief(*args):
+    og = _oyez_get(*args)
     if og:
         return {"facts": og["facts_of_the_case"], "question": og["question"], "conclusion": og["conclusion"]}
```

Comment:
Could we define a `NamedTuple` or `TypedDict`  and add a `typing` annotation to the methods just so the fields are documented?

(https://stackoverflow.com/questions/44225788/python-3-dictionary-with-known-keys-typing)